README.md

```diff
@@ -15,6 +15,8 @@
 	<p>This plugin uses <a href="https://github.com/mishoo/UglifyJS2/tree/harmony">UglifyJS v3 </a><a href="https://npmjs.com/package/uglify-es">(`uglify-es`)</a> to minify your JavaScript<p>
 </div>
 
+Note: this documentation is for the current beta. For latest stable, see https://github.com/webpack-contrib/uglifyjs-webpack-plugin/tree/v0.4.6.
```

Comment:
It's already on mentioned on L43, but could be moved here

```diff
- Note: ${text}
+ > ⚠️  ${text}
```

```diff
- https://github.com/webpack-contrib/uglifyjs-webpack-plugin/tree/v0.4.6.
+ [v0.4.6](https://github.com/webpack-contrib/uglifyjs-webpack-plugin/tree/v0.4.6)
```

Reply:
@michael-ciniawsky Thanks, updated!